pl_examples/domain_templates/generative_adversarial_net.py

```diff
@@ -85,6 +85,8 @@ def __init__(self, hparams):
         self.generated_imgs = None
         self.last_imgs = None
 
+        self.example_input_array = torch.zeros(2, hparams.latent_dim)
```

Comment:
this is very nice, just add it also to docs that it is recommended...

Reply:
sure, good idea!

Reply:
I added it to docs/debugging.rst

Reply:
what about rather implement it as property?
(not sure if you have answered it already earlier)

Reply:
no, not answered, I forgot but I am not sure what to do, since this is not read only property. Should I make setter + getter?
```python
@property
def example_input_array(self):
    return self._example_input_array

@example_input_array.setter
def example_input_array(self, array):
    self._example_input_array = array
```

Reply:
yeah that looks good

Reply:
ok made the change.

Reply:
just think that you have to 
```
def example_input_array(self):
    if ot hasattrib(self, '_example_input_array'):
        return None
    return self._example_input_array
```

Reply:
even if I set it to None in `__init__`??

Reply:
this pickle seems to be very limited, how invented that crap?

Reply:
so setting it to None in `__init__` is not enough?

Reply:
Then it is fine, missed you set up in init